js/main.js

```diff
@@ -2,17 +2,15 @@
 /* eslint-disable no-restricted-globals */
 /* eslint-disable import/extensions */
```

Comment:
- [ ] I noticed that you disable a lot **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: